SecretSanta/src/SecretSanta.Domain/Services/GiftService.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SecretSanta.Domain.Services
+{
+    public class GiftService
+    {
+        /*
+        public Gift(string title, int importance, string url, string description, User user)
+        {
+            if (title != null && url != null && description != null && user != null)
+            {
+                Title = title;
+                Importance = importance;
+                URL = url;
+                Description = description;
+                User = user;
+            }
+            else
+            {
+                throw new Exception("Gift constructor passed parameter was null");
+            }
+        }
+
+        public bool EditGift(string title, int importance, string url, string description, User user)
+        {
+            if (title != null && url != null && description != null && user != null)
+            {
+                Title = title;
+                Importance = importance;
+                URL = url;
+                Description = description;
+                User = user;
+
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        public bool DeleteGift(User user)
+        {
+            if (user != null)
+            {
+                if (user.Gifts.Contains(this))
+                {
+                    user.Gifts.Remove(this);
```

Comment:
The `List.Remove()` function will return a bool on success/failure, so you can reduce this down to probably a single line.
```suggestion
                    return user.Gifts.Remove(GiftToRemove);
```